packages/graphql-server/src/functions/types.ts

```diff
@@ -152,6 +152,13 @@ export interface GraphQLHandlerOptions {
    * Defaults to '/graphql' as this value must match the name of the `graphql` function on the api-side.
    */
   graphiQLEndpoint?: string
+
+  /**
+   * @description Allow schema introspection in production.
+   * By default, schema introspection is disabled in production. Enable this to allow introspection in production.
+   */
+  allowIntrospectionInProduction?: boolean
```

Comment:
What if just`allowIntrospection`. Optional. default is true if isDev. Then can still set true or false regardless of environment?

Reply:
Makes sense to me. Just to understand,  if `allowIntrospection` is `undefined`, we allow introspection only in dev (like right now) and we receive explicit `true` or `false`, we follow that without any check on env right?

Reply:
Correct. I think if `allowIntrospection` is undefined, it defaults to the isDev. But if true or false is set, that's its setting.

Does that seem ok?

Reply:
Yep. Did the change.